pkg/volume/csi/csi_mounter.go

```diff
@@ -188,7 +179,12 @@ func (c *csiMountMgr) SetUpAt(dir string, fsGroup *int64) error {
 	if len(fsType) == 0 {
 		fsType = defaultFSType
 	}
-
+	nodePublishCredentials := map[string]string{}
+	if csiSource.NodePublishSecretRef != nil {
+		sn := csiSource.NodePublishSecretRef.Name
+		ns := c.pod.ObjectMeta.Namespace
```

Comment:
this should be csiSource.NodePublishSecretRef.Namespace

Reply:
to protect against this type of mistake, consider passing the secret ref to getCredentialsFromSecret instead of loose name/namespace fields

Reply:
done